### PR TITLE
Updated content for Miam - valid reasons for not attending miam

### PR DESCRIPTION
**Pull request by BharadwajSairam**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PRL-5547

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/steps/c100-rebuild/miam/general-reasons/template.njk [hunk 1]
```diff
@@ -3,11 +3,8 @@
 
 {% block page_content %}
 
-<span class="govuk-caption-l">{{section}}</span>
-<h1 class="govuk-heading-xl">{{title}}</h1>
-
+<span class="govuk-heading-l">{{section}}</span>
```
Comment (vivek-sekhar) [inline src/main/steps/c100-rebuild/miam/general-reasons/template.njk:6]: Pls follow standards for captions and title in terms of markup & styles

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

File: src/main/steps/c100-rebuild/miam/general-reasons/template.njk (hunk 1 above)
Comment (BharadwajSairam) [reply src/main/steps/c100-rebuild/miam/general-reasons/template.njk:6]: But this is the expected font from the figma for the title

Comment (vivek-sekhar) [reply src/main/steps/c100-rebuild/miam/general-reasons/template.njk:6]: Please refer to other template and follow the standard markup of HTML contents

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

**Review by AlokDatta — APPROVED** [review]

File: src/main/steps/c100-rebuild/miam/general-reasons/template.njk (hunk 1 above)
Comment (vivek-sekhar) [reply src/main/steps/c100-rebuild/miam/general-reasons/template.njk]: Use the below markup.

  <h1 class="govuk-heading-l">
    <span class="govuk-caption-l">{{caption}}</span>
    {{title}}
  </h1>

Comment (vivek-sekhar) [reply src/main/steps/c100-rebuild/miam/general-reasons/template.njk]:
```
  <h1 class="govuk-heading-l">
    <span class="govuk-caption-l">{{caption}}</span>
    {{title}}
  </h1>
<div class="govuk-body"> {{content}} </div>
```

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

File: src/main/steps/c100-rebuild/miam/general-reasons/template.njk [hunk 2]
```diff
@@ -1,13 +1,6 @@
 
 {% extends "common/template.njk" %}
-
 {% block page_content %}
-
-<span class="govuk-caption-l">{{section}}</span>
-<h1 class="govuk-heading-xl">{{title}}</h1>
-
+<h1 class="govuk-heading-l govuk-!-margin-bottom-7">{{title}}</h1>
 <div class="govuk-body-m"> {{courtcommunication}} </div>
```
Comment (vivek-sekhar) [inline src/main/steps/c100-rebuild/miam/general-reasons/template.njk:5]: use <p> with class="govuk-body"

Comment (BharadwajSairam) [reply src/main/steps/c100-rebuild/miam/general-reasons/template.njk]: Changes as suggested

**Review by vivek-sekhar — APPROVED** [review]

**Pull request merged**